catalogue/webapp/tsconfig.json

```diff
@@ -30,5 +30,8 @@
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx"
-  ]
+  ],
+  "ts-node": {
+    "preferTsExts": true
```

Comment:
Will we be able to get rid of this once more flow stuff is blatted?

Reply:
Yea, as we won't have `js` extentions.